components/neeva/common/content_filtering_service.mojom

```diff
@@ -25,14 +25,20 @@ struct ContentFilterAction {
   string host;
 };
 
+// IPC: below interfaces are separate FIFOs 
```

Comment:
I think this comment block could be deleted. I think it is kind of redundant with the below. The "Implemented by the Renderer" and "Implemented by the Browser" comments seem to also tell this story more concisely.